packages/react-divider/package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "@fluentui/react-divider",
+  "version": "9.0.0-alpha.10",
+  "description": "Fluent UI component to visually separate content.",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
+    "@fluentui/react-conformance": "^1.0.0",
```

These are out of date
```suggestion
    "@fluentui/eslint-plugin": "^1.0.1",
    "@fluentui/react-conformance": "^0.2.4",
```